utils.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'net/http'
+require 'json'
+
+TRELLO_URL = 'https://api.trello.com/1/'
+APIKEY = ''
+APITOKEN = ''
```

Comment:
This should be env variables or command line options, you don't want to have tokens or credentials in VCS